pkg/agent/cniserver/ipam/testing/mock_ipam.go

```diff
@@ -1,4 +1,4 @@
-// Copyright 2019 Antrea Authors
+// Copyright YEAR Antrea Authors
```

Comment:
?

Reply:
I am not sure how this happened, Ill change it back

Reply:
Whenever a run make codegen it switches back to YEAR

Reply:
Not able to reproduce locally. But I have observed that if I manually set it to YEAR and commit that change, then it will leave it as YEAR. Even if you don't explicitly set it to YEAR, that could happen if the script failed halfway because of some in correct changes.